### ISSUE TITLE
CLI command "intfstat -D" fails when counters dump file does not exist

### ISSUE THREAD

**Issue by yvolynets-mlnx** [Azure/sonic-utilities#749]
**Description**
CLI command "intfstat -D" fails when counters dump file does not exist.

**Steps to reproduce the issue**
1. intfstat -D
2. intfstat -D

**Describe the results you received**
```
root@arc-switch1041:/home/admin# intfstat -D
Traceback (most recent call last):
  File "/usr/bin/intfstat", line 346, in <module>
    main()
  File "/usr/bin/intfstat", line 272, in main
    for file in os.listdir(cnstat_dir):
OSError: [Errno 2] No such file or directory: '/tmp/intfstat-0'
root@arc-switch1041:/home/admin# intfstat -D
Traceback (most recent call last):
  File "/usr/bin/intfstat", line 346, in <module>
    main()
  File "/usr/bin/intfstat", line 272, in main
    for file in os.listdir(cnstat_dir):
OSError: [Errno 2] No such file or directory: '/tmp/intfstat-0'
```

**Expected results**
Handle case when counters file does not exist.

**Output of `show version`**

```
root@arc-switch1041:/home/admin# show version

SONiC Software Version: SONiC.HEAD.128-b25ec7d5
Distribution: Debian 9.11
Kernel: 4.9.0-9-2-amd64
Build commit: b25ec7d5
Build date: Wed Nov 20 03:40:41 UTC 2019
Built by: johnar@jenkins-worker-4

Platform: x86_64-mlnx_msn2740-r0
HwSKU: ACS-MSN2740
ASIC: mellanox

```

**Comment by msosyak** [comment]
The fix is already created #744

**Comment by mykolaf** [comment]
@msosyak It's the same issue, but in intfstat.
